packages/data-connect/src/network/transport/rest.ts

```diff
@@ -34,6 +34,7 @@ export class RESTTransport implements DataConnectTransport {
   private _serviceName: string;
   private _accessToken: string | null = null;
   private _authInitialized = false;
+  private _lastToken: string | null = null;
```

Comment:
I just realized I forgot to check for a changed token in Android! I'll go ahead and implement that.